Gemfile

```diff
@@ -5,6 +5,8 @@ ruby '2.4.1'
 
 gem 'rails', '~> 5.2.0.rc1'
 gem 'pg', '>= 0.18', '< 2.0'
+gem 'dotenv-rails'
```

Comment:
Is this gem/library used in the solution you chose?

Reply:
Yes.

Reply:
So what is the rest of the changes introduced in this PR actually do?